modules/common/src/main/scala/surge/internal/health/HealthSignalStreamProvider.scala

```diff
@@ -85,7 +89,37 @@ trait HealthSignalStreamProvider {
    * @return
    *   HealthSignalBus
    */
-  def bus(): HealthSignalBusInternal = {
-    signalBus
+  def bus(): HealthSignalBusInternal = signalBus
+}
+
+class NullHealthSignalStream(config: HealthSignalBusConfig, bus: HealthSignalBusTrait, override val actorSystem: ActorSystem) extends HealthSignalStream {
+
+  override def signalHandler: SignalHandler = (_: HealthSignal) =>
+    Try {
+      Done
```

Comment:
Slight optimization you can do `Success(Done)` here

Reply:
done